### PR TITLE
New dominian uniform, capes and berets sprites + new accessories.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Adds a whole lot of new sprites for most dominian uniforms, capes and berets. Also adds the lyodii amulet, retired TIC investigator cards, and the tribunalist hospital cape. Sprites by DasFox and Azlan.

### CODE REVIEW

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/clothing/factions/dominia.dm [hunk 1]
```diff
@@ -656,6 +663,37 @@
 	icon_state = "dominia_amulet"
 	contained_sprite = TRUE
 
+/obj/item/clothing/accessory/dominia/lyodii
+	name = "lyodic tribunal necklace"
+	desc = "An amulet depicting the three animal aspects of Lyodic Tribunalism upon a bone plate. \
+			A prominent symbol of the Lyodic take on the religion, worn by both its shamans an followers."
```
Comment (TheSleepyCatmom) [inline code/modules/clothing/factions/dominia.dm:669]:
```suggestion
			A prominent symbol of the Lyodic take on the religion, worn by both its shamans and followers."
```

**Review by atteria — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/items/suit.dm [hunk 2]
```diff
@@ -290,6 +290,7 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["tribunalist medical cape"] = /obj/item/clothing/accessory/poncho/dominia_cape/hospital
```
Comment (DreamySkrell) [inline code/modules/client/preference_setup/loadout/items/suit.dm:293]: This should probably be restricted to medical only. Same as how the "tribunalist medical beret" is currently.

Comment (NonQueueingMatt) [reply code/modules/client/preference_setup/loadout/items/suit.dm:293]: I don't really want to bloat the loadout buttons with a new cape

Comment (DreamySkrell) [reply code/modules/client/preference_setup/loadout/items/suit.dm:293]: I think it should be restricted, and consistent with other items.
Unfortunately, I think the bloat is already here, and the loadout will not get any less messy anytime soon.
(Tags would potentially help with the bloat, but... Yeah.)

Comment (NonQueueingMatt) [reply code/modules/client/preference_setup/loadout/items/suit.dm:293]: > I think it should be restricted, and consistent with other items. We should not be cutting corners, cause like, I think the bloat is already here, and the loadout will not get any less messy anytime soon. (Tags would potentially help with the bloat, but... Yeah.)

I just don't think this is a problem at all but ok

File: html/changelogs/mattatlasdasfox-freeze.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a whole lot of new sprites for most dominian uniforms, capes and berets. "
```
Comment (SleepyGemmy) [inline html/changelogs/mattatlasdasfox-freeze.yml:41]:
```suggestion
  - rscadd: "Added a whole lot of new sprites for most dominian uniforms, capes and berets."
```

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**